packages/usdk/lib/chat.mjs

```diff
@@ -7,12 +7,20 @@ import {
   join,
   leave,
 } from '../util/connect-utils.mjs';
+import { warnIfAgentsUseNewerSDKVersion } from './version.mjs';
 
 //
 
 export const chat = async (args, opts) => {
   // console.log('got chat args', args);
   const agentSpecs = await parseAgentSpecs(args._[0]);
+
+  try {
+    await warnIfAgentsUseNewerSDKVersion(agentSpecs, opts);
```

Comment:
Probably shouldn't wait here as it will slow down the chat starting.

Reply:
Maybe cache it for a period and don't do it during chat, but on any `usdk` command.